todo.py

```diff
@@ -1,11 +1,31 @@
 import json
+import os
+from datetime import datetime
+
+def get_last_modified_timestamp(file_path):
+    try:
+        timestamp = os.path.getmtime(file_path)
+        return datetime.fromtimestamp(timestamp).strftime("%Y-%m-%d %H:%M:%S")
+    except FileNotFoundError:
+        return None
+    
+def load_tasks():
+    file_path = "tasks.json"
+    with open(file_path, "r") as file:
+        tasks = json.load(file)
```

Comment:
you should handle the exception if file not read correctly
```suggestion
 
    def load_tasks():
        file_path = "tasks.json"
        try:
            with open(file_path, "r") as file:
                tasks = json.load(file)
        except (FileNotFoundError, json.JSONDecodeError) as e:
            print(f"Error loading tasks: {e}")
            tasks = []  # or handle as appropriate
        return tasks
```